### DISCUSSION TITLE
Executing EraseWorkingFiles without preparing leads to an access violation

### DISCUSSION THREAD

**Discussion by tozGitHub** [WinEdt-Team/WinEdt-Support#35] [2.2 Issues, Bugs and Problems]
Dear Alex, dear community

Executing ```EraseWorkingFiles;``` without any preparation (that is just executing only this command and nothing else) gives an access violation:
![8e3c1094faaa9ee52c3754380cc28ef0_MD5](https://github.com/WinEdt-Team/WinEdt-Support/assets/52357589/8c3a4eba-109d-42d9-b472-4ca6acb57f30)

WinEdt version:  64-bit and Build: 20240319  (v. 1.2) - 64-bit (portable)
Windows version: 10.0.22631 Build 22631.

Best,
Thomas

**Comment by Alex-Simonic** [comment]
You finally found a real "bug"! Of course, the call without any initialization does not make sense but AV is not a proper way to tell you that. In the next version you will get:

```
    WarningMsg('File Items are not initialized...'+CRLF+
               'There is nothing to delete!');
```
